.changeset/chilled-chairs-ring.md

```diff
@@ -0,0 +1,6 @@
+---
+"@commercetools-uikit/collapsible-panel": patch
```

Comment:
I think we should consider these `feat`, since we're adding a new prop to the component 🙂